### PR TITLE
[GH-1144] Helper script for importing Data Repo data into Terra

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose

Provide an example of how to get data from a snapshot in the Data Repo into a Terra workspace:
https://broadinstitute.atlassian.net/browse/GH-1144

### Changes

Given a data repo snapshot, this script will:
1. Query the snapshot sample table in BigQuery
2. Save the table contents as a TSV file and return a list of the samples/entities formatted for WFL. This script uses the `datarepo_row_id` as the unique id (AKA the "entity id") of the sample in Terra.
3. Upload the TSV file to the specified Terra workspace data table

Note that the default snapshot uses the datarepo integration test dataset, since we do not have a test arrays dataset yet.

### CODE REVIEW

File: ops/data_repo_terra/import_snapshot_data.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    import_url = f'{terra_url}/api/workspaces/{terra_workspace}/flexibleImportEntities'
+    scopes = ['email', 'openid', 'profile']
+    credentials = get_service_account_credentials(service_account_path, scopes)
+    headers = {'Authorization': f'Bearer {credentials.token}'}
+    with open (tsv_file, 'rb') as f:
+        contents = f.read()
+    response = requests.post(import_url,
+                             headers=headers,
+                             files={'entities': io.BytesIO(contents),
+                                    'type': 'text/tab-separated-values'})
```
Comment (rexwangcc) [inline ops/data_repo_terra/import_snapshot_data.py:61]:
```suggestion
                                    'type': 'text/tab-separated-values'})
    response.raise_for_status()
```

Comment (rexwangcc) [reply ops/data_repo_terra/import_snapshot_data.py:61]: nit?

File: ops/data_repo_terra/import_snapshot_data.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    print(f'The following samples have been uploaded to {terra_workspace}:')
+    for sample in entities:
+        print(sample)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Import Data Repo snapshot data into a Terra workspace.",
+                                     usage="%(prog)s [-h] [DATAREPO_SNAPSHOT TERRA_URL TERRA_WORKSPACE SERVICE_ACCOUNT_PATH] [...]")
+    parser.add_argument("--datarepo_snapshot",
+                        default="zerosnapshot",
```
Comment (rexwangcc) [inline ops/data_repo_terra/import_snapshot_data.py:78]: should it be `wflsnapshot`, or zero means a different thing here?

**Review by rexwangcc — APPROVED** [review]
Looks great! Thanks for sharing this helper script!

File: ops/data_repo_terra/import_snapshot_data.py (hunk 2 above)
Comment (samanehsan) [reply ops/data_repo_terra/import_snapshot_data.py:79]: The original dataset had `zero` in it instead of `wfl` so I used it here for consistency!

**Pull request merged**